docs/mastering-plone-5/instructions_plone5/index.md

```diff
@@ -0,0 +1,382 @@
+2019-09-05 20:11:07,151 WARNING [plone.behavior:172][MainThread] Specifying 'for' in behavior 'Tiles' if no 'factory' is given has no effect and is superfluous.
+2019-09-05 20:11:08,353 WARNING [PrintingMailHost:30][MainThread] Hold on to your hats folks, I'm a-patchin'
+2019-09-05 20:11:08,353 WARNING [PrintingMailHost:124][MainThread]
+
+******************************************************************************
+
+Monkey patching MailHosts to print e-mails to the terminal.
+
+This is instead of sending them.
+
+NO MAIL WILL BE SENT FROM ZOPE AT ALL!
+
+Turn off debug mode or remove Products.PrintingMailHost from the eggs
+or remove ENABLE_PRINTING_MAILHOST from the environment variables to
+return to normal e-mail sending.
+
+See https://pypi.python.org/pypi/Products.PrintingMailHost
+
+******************************************************************************
+
+2019-09-05 20:11:08,390 INFO    [Zope:45][MainThread] Ready to handle requests
+Starting server in PID 30620.
+Serving on http://0.0.0.0:8080
+```
+
+If the output says `Serving on http://0.0.0.0:8080` then you are in business.
+
+If you point your browser at <http://localhost:8080> you see that Plone is running.
+
+```{figure} _static/instructions_plone_running.png
+:alt: A running Plone instance.
+:scale: 50 %
+
+A running plone instance.
+```
+
+There is no Plone site yet - we will create one in chapter 6.
+
+Now you have a working Plone site up and running and can continue with the next chapter.
+
+You can stop the running instance anytime using {kbd}`ctrl + c`.
+
+```{warning}
+If there is an error message you should either try to fix it or use vagrant and continue in this chapter.
+```
+
+(plone5-instructions-vagrant-label)=
+
+## Installing Plone with Vagrant
+
+We use a virtual machine (Ubuntu 18.04) to run Plone during the training.
+
+We rely on [Vagrant](https://www.vagrantup.com) and [VirtualBox](https://www.virtualbox.org) to give the same development environment to everyone.
+
+[Vagrant](https://www.vagrantup.com) is a tool for building complete development environments.
+
+We use it together with Oracle’s [VirtualBox](https://www.virtualbox.org) to create and manage a virtual environment.
+
+(plone5-install-virtualbox)=
+
+### Install VirtualBox
+
+Vagrant uses Oracle’s VirtualBox to create virtual environments.
+
+Here is a link directly to the download page: <https://www.virtualbox.org/wiki/Downloads>.
+
+We use VirtualBox 6.0.x
+
+(plone5-instructions-configure-vagrant-label)=
+
+### Install and configure Vagrant
+
+Get the latest version from <https://developer.hashicorp.com/vagrant/install> for your operating system and install it.
+
+Now your system has a command {command}`vagrant` that you can run in the terminal.
+
+First, create a directory in which you want to do the training.
+
+```{warning}
+If you already have a {file}`training` directory because you followed the **Installing Plone without vagrant** instructions above,
```

Comment:
```suggestion
If you already have a {file}`training` directory because you followed the {ref}`plone5-instructions-no-vagrant-label` instructions above,
```